src/Data/Quantity.purs

```diff
@@ -136,12 +137,16 @@ toStandard (num .*. du) =
   case U.toStandardUnit du of
     Tuple du' conversion → (conversion * num) ..* du'
 
--- | Attempt to simplify the unit of a quantity.
+-- | Attempt to simplify the unit of a quantity, except angles.
```

Comment:
To be precise, the docstring should be something like:

> Attempt to simplify the unit of a quantity. Quantities with units that be completely cancelled will be reduced to scalar values (except for angle-units)

.. and similar for the `fullSimplify'` function below.